gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -965,8 +967,21 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
       g_param_spec_string ("output-combination", "output tensor(s) combination",
           "Select the output tensor(s) from the input tensor(s) and/or model output",
           "", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_SHARED_TENSOR_FILTER_KEY,
+      g_param_spec_string ("shared-tensor-filter-key",
```

Comment:
simply 'sharing-key' ?

Reply:
Through the previous discussion, to let users know easily, `shared-tensor-filter-key` has been recommended.